### ISSUE TITLE
Model errors should be visible in the UI

### ISSUE THREAD

**Issue by hannobraun** [hannobraun/Fornjot#30]
When a model fails to compile, an error is shown on the command-line, but not in the UI itself. We should at least show a message, stating that an error has occurred and that more information is available in the CLI output. It would be even better, if the full error was displayed in the UI itself.

Marking as https://github.com/hannobraun/Fornjot/labels/status%3A%20blocked, as it would be better to figure out #116 first.

**Comment by hannobraun** [comment]
It seems that the *Milestones* feature is meant to track milestones as a series, one happening after another (like releases, for example). I'm using the feature differently, to define goal posts that can be worked on and completed in parallel.

This issue should belong to both [*Basic Usability*](https://github.com/hannobraun/Fornjot/milestone/2) and [*Modeling Convenience*](https://github.com/hannobraun/Fornjot/milestone/4), but I can only add it to one. I'm leaving it with *Basic Usability*, as that one seems more likely to be completed first.

**Comment by hannobraun** [comment]
Closing in favor of #856.
